Tevling/Pages/Profile.razor

```diff
@@ -1,44 +1,103 @@
 @page "/profile"
 
-<h1>Profile</h1>
-
-<h5 class="mt-3">Hello, @Athlete.Name!</h5>
-<p>
-    You joined Tevling @CreatedTime.
-    You're currently following @Athlete.Following?.Count <a href="/athletes">athletes</a> and
-    have @Athlete.Followers?.Count followers.
-</p>
-
-<h4 class="mt-4">Import activities from Strava</h4>
-<p>Import last 30 days of activities to get started!<br />
-<em style="font-size: 0.9rem;">(This can only be done once)</em></p>
-<p>
-    <button class="btn btn-primary" @onclick="Import" disabled="@(Importing || Athlete.HasImportedActivities)">
-        @if (Athlete.HasImportedActivities)
-        {
-            <text>Import complete</text>
-        }
-        else if (Importing)
-        {
-            <text>Importing...</text>
-        }
-        else
-        {
-            <text>Import activities</text>
-        }
-    </button>
-</p>
-@if (!string.IsNullOrEmpty(ImportResult))
-{
-    <p>@ImportResult</p>
-}
-
-<h4 class="mt-4">Sign out</h4>
-<p>If using a public/shared computer, please remember to sign out.</p>
-<p><a class="btn btn-secondary" href="/auth/logout">Sign out</a></p>
-
-<hr class="my-4" />
-<h4>Delete account</h4>
-<p class="text-danger">Delete user account and deauthorize application. All data will be removed.<br />
-<em>NB! This cannot be undone!</em></p>
-<p><a class="btn btn-danger" href="/auth/logout?deauthorize=true">Forget me</a></p>
+<h1 class="title mb-5">Profile</h1>
+<div class="profile d-flex flex-row column-gap-2 mb-5">
+    <Avatar Url="@Athlete.ImgUrl" Size="50" />
+    <div class="d-flex flex-column">
+        <span class="profile-name">@Athlete.Name</span>
+        <span class="member-since">Member since @CreatedTime</span>
+    </div>
+</div>
+
+<div class="d-flex flex-row flex-wrap">
+    <div class="profile-info d-flex flex-column row-gap-5">
+        <div class="followers">
+            <h2 class="mb-3">Followers</h2>
+            @if (Athlete.Followers?.Count == 0 || Athlete.Followers is null)
+            {
+                <span>You currently have no followers 😕</span>
+            }
+            else
+            {
+                <div class="d-flex flex-column row-gap-3">
+                @foreach (var follower in Athlete.Followers)
+                    {
+                        <AthleteCard Athlete="@follower" IsFollowing="Athlete.IsFollowing(follower.Id)" />
```

Comment:
If we're not able to toggle following here, should we disable the user interaction (i.e. hover effect)?

Reply:
I mean we're not handling the `FollowingStatusChanged` event here. Either it should be handled, or the component should be rendered differently to disable the hover effect.